docs/src/pages/components/tables/DataTable.tsx

```diff
@@ -2,14 +2,14 @@ import * as React from 'react';
 import { DataGrid, GridColDef, GridValueGetterParams } from '@material-ui/data-grid';
 
 const columns: GridColDef[] = [
-  { field: 'id', headerName: 'ID', width: 70 },
-  { field: 'firstName', headerName: 'First name', width: 130 },
-  { field: 'lastName', headerName: 'Last name', width: 130 },
+  { field: 'id', headerName: 'ID', width: 90 },
+  { field: 'firstName', headerName: 'First name', width: 150 },
+  { field: 'lastName', headerName: 'Last name', width: 150 },
   {
     field: 'age',
     headerName: 'Age',
     type: 'number',
-    width: 90,
+    width: 110,
```

Comment:
I would propose we revert these changes so we can keep a denser display of information. It seems to be a regression in mui-x that we can easily solve (display none instead of visibility hidden).

Reply:
> I would propose we revert these changes so we can keep a denser display of information. It seems to be a regression in mui-x that we can easily solve (display none instead of visibility hidden). cc @m4theushw who last worked on this logic.

Alright, will revert.